adapters/repos/db/inverted/bm25_searcher.go

```diff
@@ -308,6 +308,10 @@ func (b *BM25Searcher) getTopKObjects(topKHeap *priorityqueue.Queue[any],
 			for j, result := range results {
 				if termIndice, ok := indices[j][res.ID]; ok {
 					queryTerm := result.queryTerm
+					if len(result.data) <= termIndice {
+						b.logger.Warnf("Skipping object explanation in BM25: termIndice %v is out of range for queryTerm %v, len() %d, res.ID %v", termIndice, queryTerm, len(result.data), res.ID)
```

Comment:
can we change `termIndice` to `term indice` and `queryTerm` to `query term`, so that we would not expose our internal variable names in the logs, wdyt?